### DISCUSSION TITLE
How to install on MacBookPro with M1 chip?

### DISCUSSION THREAD

**Discussion by nidabouzid** [atoti/atoti#595] [Q&A] [answered]
Hi
on a fresh new laptop when I follow instructions for atoti installation I end up having this version
`  atoti              /noarch::atoti-0.4.3-74d673a`
I was expecting 0.6.6, is that normal ?

**Comment by tibdex** [comment]
Hi Nida, this depends on https://github.com/atoti/jdk4py/issues/52. We'll see what we can do. We definitely want to support Apple silicon since I expect more and more technical users to use it.

**Reply by rcolle** [reply]
Can he force the installation of 0.6.6 and then provide its own JDK?

**Reply by tibdex** [reply]
Not easily but it might be done with [`pip install --no-deps`](https://stackoverflow.com/a/54411695) to avoid installing `jdk4py` (but still install `atoti-core` and the other required deps) and then set the `JAVA_HOME` environment variable to a valid JDK.

**Reply by nidabouzid** [reply]
I just forced to 0.6.6 it did not work, did not try yet [pip install --no-deps](https://stackoverflow.com/a/54411695)

**Reply by nidabouzid** [reply]
can we inform developers in our installation instruction about this issue ?

**Reply by tibdex** [reply]
I worked on https://github.com/atoti/jdk4py/pull/54, can you try again?

**Reply by nidabouzid** [reply]
works now , thanks !

**Comment by tibdex** [comment, marked as answer]
Since https://github.com/atoti/jdk4py/pull/54, atoti can now be installed on Apple Silicon.
